sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeBusinessCardsOptions.cs

```diff
@@ -33,8 +33,8 @@ public RecognizeBusinessCardsOptions()
         public FormContentType? ContentType { get; set; }
 
         /// <summary>
-        /// Locale value. Supported locales include: en-AU, en-CA, en-GB, en-IN, en-US.
+        /// Set the locale information for the business card.
```

Comment:
Maybe `Sets the locale...` (like in `This property sets the locale...`). I always start these descriptions using the third person. Not sure if it's a standard or just a matter of preference, so it's up to you.

Reply:
In case you decide to change it, same comment in `RecognizeInvoiceOptions.Locale` and `RecognizeReceiptsOptions.Locale`.